torch_xla/experimental/spmd_fully_sharded_data_parallel.py

```diff
@@ -44,10 +46,12 @@ class SpmdFullyShardedDataParallel(nn.Module):
   def __init__(
       self,
       module: nn.Module,
+      *,
       mesh: Optional[spmd.Mesh] = None,
       shard_output: Optional[Callable] = None,
       auto_wrap_policy: Optional[Callable] = None,
       auto_wrapper_callable: Optional[Callable] = None,
+      extra_data_axis: Optional[str] = None,
```

Comment:
What do you think of calling it `replica_axis` instead of `extra_data_axis`?

Reply:
I think replica_axis is too tied to the underneath technology while users may only be familiar with data paralle, fsdp, and tensor parallel.